src/style-value.js

```diff
@@ -16,34 +16,55 @@
 
   function StyleValue() {}
 
-  // TODO: Add support for cssString being a sequence.
-  StyleValue.parse = function(property, cssString) {
```

Comment:
I preferred calling this cssString rather than value here. Please put that back :)